### PR TITLE
Security patch

### LINKED ISSUE #78: Uninitialized Memory Exposure
Uninitialized Memory Exposure

Medium severity
Vulnerable module: concat-stream

Introduced through: brfs@1.4.3 › static-module@1.3.1 › concat-stream@1.4.10

https://snyk.io/vuln/npm:concat-stream:20160901

### PR DESCRIPTION
**Pull request by hubdotcom**
Fixes #78 via static-module and concat-stream update . Bump version

### CODE REVIEW

**Comment by mattdesl** [comment]
I don't think this is necessary because of the carets.

**Comment by hubdotcom** [comment]
It gets flagged by security testing software so ideally these should be updated to patched versions avoid false positives 
https://snyk.io/vuln/npm:concat-stream:20160901

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "brfs",
-  "version": "1.4.3",
+  "version": "1.4.4",
```
Comment (stevemao) [inline package.json]: Can you revert this line? otherwise LGTM

Comment (hubdotcom) [reply package.json]: done

**Pull request merged**